content/troubleshooting/common-issues.md

```diff
@@ -92,3 +92,24 @@ When building for iOS, the build gets stuck after showing `Xcode build done` in
 * https://github.com/flutter/flutter/issues/35988
 
 This issue is known to be fixed on the `master` channel.
+
+## iOS build error with `Provisioning profile`
+
+**Description**:
+
+This issue takes place when trying to use an outdated  Provisioning profile that do not include 
+Associated Domains. 
+
+**Log output**: 
+
+❌ error: Provisioning profile "CodeMagic" doesn't support the Associated Domains capability. (in target 'Runner' from project 'Runner') 
+❌ error: Provisioning profile "CodeMagic" doesn't include the com.apple.developer.associated-domains entitlement. (in target 'Runner' from project 'Runner')
+
+**Solution**: 
+
+ Log in to Apple Developer Account and verify :
```

Comment:
Log in to Apple Developer Account and:
  - Verify that you have the correct bundle ID.  
  - If the Associated Domain Entitlement is missing, add it from here.
  - Update the provisioning profile and use it to configure your project.